Framework/PythonInterface/mantid/api/src/Exports/ExperimentInfo.cpp

```diff
@@ -81,5 +82,9 @@ void export_ExperimentInfo() {
       .def("detectorInfo", &ExperimentInfo::detectorInfo,
            return_value_policy<reference_existing_object>(), args("self"),
            "Return a const reference to the :class:`~mantid.api.DetectorInfo` "
+           "object.")
+      .def("spectrumInfo", &ExperimentInfo::spectrumInfo,
```

Comment:
I think this is currently in the child class export `MatrixWorkspace`, so if we add it here in the base, remove it in the child.